lib/msal-browser/docs/caching.md

```diff
@@ -58,7 +58,7 @@ To faciliate efficient token acquisition while maintaining a good UX, MSAL cache
     - previous failed request 
     - performance data
 
-> :bulb: Temporary cache entries will always be stored in session storage or in memory. MSAL will fallback to memory storage if local/session storage is not available.
+> :bulb: Temporary cache entries will always be stored in session storage or in memory unless overridden by the user with the `temporaryCacheLocation` cache option. MSAL will fallback to memory storage if local/session storage is not available.
```

Comment:
I think we will also want to clearly document the potential problems with overriding the temporary cache location. Specifically when choosing localStorage interaction in more than one tab/window will not be supported and you may receive interaction_in_progress errors unexpectedly. 

Please describe the scenario where this should be overridden. This is an escape hatch, not a fully supported feature.

Reply:
Agreed. I documented this more explicitly now.